### ISSUE TITLE
What's the state?

### ISSUE THREAD

**Issue by kim366** [ion-lang/ion#3]
Hi, this language looks very impressive, simplifying a lot of stuff, but preserving JS syntax. I am wondering, with 3 years since the last commit if this can be used in production or if it is unstable and was never finished

**Comment by whitelizard** [comment]
> Hi, this language looks very impressive, simplifying a lot of stuff, but preserving JS syntax.

I agree! Found this a week ago and think it would be a great project to see grow. It would need stability, and linting, to be really interesting.

> I am wondering, with 3 years since the last commit if this can be used in production or if it is unstable and was never finished

It has a note on that at the bottom of the README.
